domains/*.polygon.is-not-a.dev.json

```diff
@@ -7,7 +7,8 @@
     },
 
     "record": {
-        "CNAME": "gloomyfm.github.io"
+        "NS": ["ns1.hostry.com", "ns2.hostry.com", "ns3.hostry.com", "ns4.hostry.com"],
```

Comment:
Please provide a specific use case for NS records and explain the reason for them.

Reply:
I would need to use NS records because i don't want to go through all of the trouble of making a new request every time i want to change something and also because i want to have control over my domain.